Dockerfile

```diff
@@ -30,27 +30,19 @@ RUN dotnet publish -a $TARGETARCH -c Release -o /app --no-restore --self-contain
 # Final stage/image
 FROM mcr.microsoft.com/dotnet/runtime:8.0 AS runtime
 
-# Determine the distribution and install libaio1
```

Comment:
why was this part removed in the PR?

Reply:
The default .NET 8 base image is [Debian](https://github.com/dotnet/dotnet-docker/discussions/5709), and this will remain the case at least until .NET 10. There is no reason to keep that `if statements` in Dockerfile.